PathKit/Sources/Angle.swift

```diff
@@ -27,6 +27,27 @@ public struct Angle: Equatable {
         }
     }
 
+    /// True if the angle is positive, false otherwise.
+    public var isPositive: Bool {
+        return 0.0 < radians
+    }
+
+    /// A positive normalized angle between 0 and 2 * .pi.
+    public var positiveNormalized: Angle {
+        let twoPI = 2 * Angle.pi
+        let angle = Angle(radians: radians.truncatingRemainder(dividingBy: twoPI.radians))
+        return isPositive ? angle : angle + twoPI
+    }
+
+    /// Returns the angular distance between two angles as an angle ranging between 0 and 180 degrees.
+    /// Eg: distance between 1 and -360 degrees is 1 degree.
```

Comment:
Let's format this comment better. Don't be afraid of some empty lines.

Reply:
done